src/locale/locales/es/messages.po

```diff
@@ -1695,21 +1695,21 @@ msgstr "Copia el enlace a la lista"
 #: src/view/com/util/forms/PostDropdownBtn.tsx:452
 #: src/view/com/util/forms/PostDropdownBtn.tsx:461
 msgid "Copy link to post"
-msgstr "Copia el enlace a la post"
+msgstr "Copiar el enlace hacia el post"
 
 #: src/components/dms/MessageMenu.tsx:110
 #: src/components/dms/MessageMenu.tsx:112
 msgid "Copy message text"
-msgstr ""
+msgstr "Copiar mensaje de texto"
```

Comment:
```suggestion
msgstr "Copiar texto del mensaje"
```

You are copying the _text_ of the _message_, in the same sense as the string below